lua/gitlab/state.lua

```diff
@@ -33,6 +33,32 @@ M.settings = {
     resolved = "✓",
     unresolved = "",
   },
+  discussion_sign = {
+    -- See :h sign_define for details about sign configuration.
+    enabled = true,
+    text = "💬",
+    linehl = nil,
+    texthl = nil,
+    culhl = nil,
+    numhl = nil,
+    priority = 20,
+    helper_signs = {
+      -- For multiline comments the helper signs are used to indicate the whole context
+      -- Priority of helper signs is lower than the main sign (-1).
+      enabled = true,
+      start = "↑",
+      mid = "|",
+      ["end"] = "↓",
+    },
+  },
+  discussion_diagnostics = {
```

Comment:
Sort of the same note as before, diagnostics are a pretty niche part of the Neovim experience, if we can add a short explanation into the README that would be great. Otherwise I don't know that it will be very user-friendly, and doubt that folks will actually use it.

Reply:
I updated the docs, please check if it is sufficient.